### DISCUSSION TITLE
Code review IM-11

### DISCUSSION THREAD

**Discussion by Nailgy** [HowProgrammingWorks/LiveQA#66] [Code review]
Offline browser game for 2 players
https://github.com/Nailgy/Deep_sea_game

**Comment by tshemsedinov** [comment]
https://github.com/Nailgy/Deep_sea_game/issues/1

**Comment by Nailgy** [comment]
Здравствуйте, хотел бы получить второй код ревью https://github.com/Nailgy/Deep_sea_game
Что изменилось с прошлого раза:
- переписаны функции в соответствии с предыдущим ревью
- созданы кнопки для взаимодействия игрока с игрой
- добавлен JS функционал на каждую из кнопок (пока что без отображения в вебе, это пока в процессе).
Наиболее интересующий вопрос - как грамотно объединить использование функций, классов и взаимодействия через html и css. Пока что делал функцию main(), в которой и происходит непосредственное взаимодействие и игрой, но, например, не знаю уместно ли описывать какие-то ещё функции внутри мейна. Заранее спасибо!

**Reply by Nailgy** [reply]
Я немного разобрался, начал переписывать куски кода + дебажу сейчас с console.log() чтобы понимать, исправно ли работает функционал игры. Поэтому пока что просьба про ревью на пару дней переноситься

**Comment by Nailgy** [comment]
Переписывать и дебажить закончил, но пока что по прежнему нет полноценной визуализации через html и css, пока что только через console.log(). Старался писать в парадигме ооп, поэтому хотел бы услышать рекомендации в этом плане в первую очередь, т.к. впервые с таким работаю и хотел бы на будущее знать как нужно. Ссылка та же https://github.com/Nailgy/Deep_sea_game
